Microsoft.Toolkit.Uwp.SampleApp/SamplePages/DirectWriteTextBlock/DirectWriteTextBlockCode.bind

```diff
@@ -0,0 +1,27 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls.WinRT"
+    mc:Ignorable="d">
+
+    <Grid>
+        <StackPanel Background="@[ContainerBackground:String:Black]">
+            <TextBlock Text="This is a standard XAML Textblock for comparison."
+                FontFamily="@[FontFamily:String:Segoe UI]"
+                FontWeight="@[FontWeight:String:Normal]"
+                FontStyle="@[FontStyle:Enum:FontStyle.Normal]"
+                FontStretch="@[FontStretch:Enum:FontStretch.Normal]"
+                FontSize="@[FontSize:Slider:15:1-100]" />
+            <controls:DirectWriteTextBlock
+                Text="@[Text:String:Windows Community Toolkit]"
+                FontFamily="@[FontFamily:String:Segoe UI]"
+                FontWeight="@[FontWeight:String:Normal]"
+                FontStyle="@[FontStyle:Enum:FontStyle.Normal]"
+                FontStretch="@[FontStretch:Enum:FontStretch.Normal]"
+                TextOrientation="@[TextOrientation:Enum:Orientation.Vertical]"
```

Comment:
When orientation is Horizontal, the text is centered (different when compared to the TextBlock)

Reply:
I've resolved this by setting Align to left.